uber/site_sections/staffing_admin.py

```diff
@@ -41,6 +41,74 @@ def _format_import_params(target_server, api_token):
     return (target_url, target_host, remote_api_token.strip())
 
 
+def _create_copy_department(from_department):
+    to_department = Department()
+    for field in ['name', 'description', 'solicits_volunteers', 'is_shiftless',
+                  'is_setup_approval_exempt', 'is_teardown_approval_exempt', 'max_consecutive_hours']:
+        if field in from_department:
+            setattr(to_department, field, from_department[field])
+    return to_department
+
+
+def _copy_department_roles(to_department, from_department):
```

Comment:
This _function and the ones below are just me moving code out of the import_shifts page handler I could create an bulk import page handler.